packages/grid/data-grid/src/tests/keyboard.DataGrid.test.tsx

```diff
@@ -94,6 +94,36 @@ describe('<DataGrid /> - Keyboard', () => {
   });
 
   /* eslint-disable material-ui/disallow-active-element-as-key-event-target */
+  it('should call preventDefault when using keyboard navigation', () => {
+    const event = { key: 'ArrowRight', preventDefault: spy() };
+
+    const columns = [
+      {
+        field: 'id',
+      },
+      {
+        field: 'name',
+      },
+    ];
+
+    const rows = [
+      {
+        id: 1,
+        name: 'John',
+      },
+    ];
+
+    render(
+      <div style={{ width: 300, height: 300 }}>
+        <DataGrid rows={rows} columns={columns} />
+      </div>,
+    );
+    // @ts-ignore
+    document.querySelector('[role="cell"][data-rowindex="0"][aria-colindex="0"]').focus();
+    fireEvent.keyDown(document.activeElement!, event);
+    expect(event.preventDefault.called).to.equal(true);
```

Comment:
Guys, I would really appreciate the help with this one. I've been trying to test the scroll prevention when an active element is not the root of a cell but I'm struggling with it. I tried a couple of different approaches but couldn't make it fail. In the end, I thought of testing whether the `event.preventDefault` function is being called but that always returns `false` (and the call count is 0). I'm kind of out of ideas at this point.